File: app/src/main/java/com/bq/zowi/interactors/ConnectToZowiInteractor.java
package com.bq.zowi.interactors;

import com.bq.zowi.models.ConnectionSuccessData;
import rx.Single;

/* JADX INFO: loaded from: classes.dex */
public interface ConnectToZowiInteractor {
    Single<Void> connectToZowi(String str, boolean z);

    Single<ConnectionSuccessData> connectToZowiAndRetrieveData(String str);

    Single<Void> disconnectFromZowi(String str);
}

File: zowi-core/src/main/kotlin/com/bq/zowi/usecases/ConnectToZowiInteractor.kt
package com.bq.zowi.usecases

import com.bq.zowi.models.ConnectionSuccessData
import io.reactivex.Completable
import io.reactivex.Single

interface ConnectToZowiInteractor {
    fun connectToZowi(deviceAddress: String, duplex: Boolean): Completable
    fun connectToZowiAndRetrieveData(deviceAddress: String): Single<ConnectionSuccessData>
    fun disconnectFromZowi(deviceAddress: String): Completable
}
